fixes.inc

```diff
@@ -9796,6 +9865,67 @@ native BAD_SetPlayerName(playerid, const name[]) = SetPlayerName;
 		// =====================
 		//  BEGIN: OnVehicleMod
 		// =====================
+
+
+		switch(FIXES_GetVehicleComponentType(componentid))
+		{
+			#if !FIXES_Single
```

Comment:
Don't use `#if !` with `#else`.  Swap the two branches around so the conditions are easier to read.  This is true in all coding.